src/util.js

```diff
@@ -0,0 +1,16 @@
+const url = 'https://rancid-tomatillos.herokuapp.com/api/v2'
```

Comment:
I like that we went with putting our fetch function inside of the util folder. I think it will keep our repo nice and tidy!